pkg/gather/gather_test.go

```diff
@@ -392,18 +394,18 @@ func assertMetadataOneGatherer(
 	err := json.Unmarshal(metadataBytes, &archiveMetadata)
 	assert.NoError(t, err)
 
-	for i := range archiveMetadata.StatusReports {
-		archiveMetadata.StatusReports[i].Duration = 0
+	for _, v := range archiveMetadata.StatusReports {
+		v.Duration = 0
 	}
 
 	assert.Equal(t, isGlobalObfuscationEnabled, archiveMetadata.IsGlobalObfuscationEnabled)
-	assert.ElementsMatch(t, statusReports, archiveMetadata.StatusReports)
+	assert.True(t, reflect.DeepEqual(statusReports, archiveMetadata.StatusReports))
```

Comment:
Why? As far as I see ElementsMatch also uses DeepEqual for everything except bytes, but also ignores the order of the elements.

Reply:
There was some failure AFAIK....if we keep it as array then we can keep the orginal definition I guess.